packages/clerk-js/src/ui/icons/index.ts

```diff
@@ -8,6 +8,7 @@ export { default as ArrowLeftIcon } from './arrow-left.svg';
 export { default as ArrowRightIcon } from './arrow-right.svg';
 export { default as AuthApp } from './auth-app.svg';
 export { default as Clipboard } from './clipboard.svg';
+export { default as CheckCircle } from './check-circle.svg';
```

Comment:
❓ Did we pass these SVGs from an SVG optimizer or does webpack do this for us?

Reply:
All I need was copy the generated SVG code from the Figma icons. I will check for further actions

Reply:
This is now optimized